### DISCUSSION TITLE
总是提示要位置权限，请问是什么原因。

### DISCUSSION THREAD

**Discussion by Momoooc** [constanline/XQuickEnergy#454] [Q&A/问答时间]

**Comment by kimshift** [comment]
解决方法：强制停止支付宝和芝麻粒软件。删除xqe文件夹，再重新打开支付宝，即可修复。不行再删除几遍xqe文件夹直到正常。

**Comment by Zhangfanyu1222** [comment]
用管理器在Download目录下创建useMedia文件夹，在Android/media文件夹下创建com.eg.android.AlipayGphone文件夹，然后在新建的文件夹下，把原本Download文件夹下的xqe文件夹复制过来，或者直接删除xqe文件夹（删除文件夹记得备份配置文件）
